### PR TITLE
short license

### PR DESCRIPTION
**Pull request by dgarus**
closes: #80

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+MIT License
```
Comment (Swizbiz) [inline LICENSE.txt:1]: Full name is **The MIT License (MIT)** and short is just **MIT**
I think we should use _The MIT License (MIT)_

**Review by Swizbiz — CHANGES_REQUESTED** [review]
Thanks!
Please check my comment.

File: LICENSE.txt (hunk 1 above)
Comment (dgarus) [reply LICENSE.txt]: done

**Review by Swizbiz — APPROVED** [review]
Many thanks!

**Pull request merged**